pkgs/development/libraries/libserdes/default.nix

```
@@ -28,6 +29,14 @@ stdenv.mkDerivation rec {
 
   makeFlags = [ "GEN_PKG_CONFIG=y" ];
 
+  patches = [
+    # Fix compatibility with Avro master branch
+    (fetchpatch {
+      url = "https://github.com/confluentinc/libserdes/pull/17.patch";
```

The hash of that URL could change in the future. Using the follow is advised. 

```suggestion
      url = "https://github.com/confluentinc/libserdes/commit/d7a355e712ab63ec77f6722fb5a9e8056e7416a2.patch";
```

---

Updated. But i guess this only works because there is only one commit.